include/nbl/builtin/hlsl/glsl_compat/core.hlsl

```diff
@@ -222,7 +229,7 @@ T bitfieldInsert(T base, T insert, uint32_t offset, uint32_t bits)
 template<typename T>
 T bitfieldReverse(T value)
 {
-    return spirv::bitFieldReverse<T>(value);
+    return spirv::bitReverse(value);
```

Comment:
I know that GLM is crap and can't have the template parameter passed directly, but the spirv intrinsic can

Reply:
passed to @Przemog1